src/server.py

```diff
@@ -86,6 +100,12 @@ def fade_out(self):
             return self.metadata["fade-out"]
         return None
 
+    @property
+    def speed(self):
```

Comment:
What does this property do, now that we have pitch and rate? I think we can remove it as well as the seek events, and just let soundtouch handle everything.